### PR TITLE
Insiders Cloud

### PR DESCRIPTION
**Pull request by tillbier**
---
- [x] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

### CODE REVIEW

**Comment by vmanoharas** [comment]
Hello @tillbier, 

Thank you for submitting your connector, kindly remove proposal and follow the instructions provided in the below link and accept the license/cla agreement. 

https://learn.microsoft.com/en-us/connectors/custom-connectors/certification-submission-ip

**Comment by tillbier** [comment]
@microsoft-github-policy-service agree company="Insiders Technologies GmbH"

File: certified-connectors/Insiders Cloud/readme.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Insiders Cloud connector
+Insiders Cloud is used to automate document processing. This Powerautomate app enables customers of the Insiders Cloud to analyze their documents.
```
Comment (vmanoharas) [inline certified-connectors/Insiders Cloud/readme.md]: Please refer the below link and update all required fields. 

https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md

**Review by vmanoharas — COMMENTED** [review]
Hello @tillbier, 

Please resolve review comments.

File: certified-connectors/Insiders Cloud/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,441 @@
...
+        },
+        "pageNumber": {
+          "type": "integer",
+          "format": "int32"
+        }
+      }
+    },
+    "FileType": {
+      "type": "string",
+      "description": "",
```
Comment (vmanoharas) [inline certified-connectors/Insiders Cloud/apiDefinition.swagger.json]: I would suggest you keep all parameters descriptions short to fit in the text box. please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#parameters).

File: certified-connectors/Insiders Cloud/apiDefinition.swagger.json [hunk 3]
```diff
@@ -0,0 +1,441 @@
...
+          "$ref": "#/definitions/FieldVerifierAction"
+        },
+        "checkState": {
+          "$ref": "#/definitions/FieldCheckState"
+        }
+      }
+    },
+    "FieldType": {
+      "type": "string",
+      "description": "",
```
Comment (vmanoharas) [inline certified-connectors/Insiders Cloud/apiDefinition.swagger.json]: I would suggest you keep all parameters descriptions short to fit in the text box. please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#parameters).

File: certified-connectors/Insiders Cloud/apiDefinition.swagger.json [hunk 4]
```diff
@@ -0,0 +1,441 @@
...
+        "pageIndex": {
+          "type": "integer",
+          "format": "int32"
+        }
+      }
+    },
+    "FieldVerifierAction": {
+      "x-nullable": true,
+      "type": "string",
+      "description": "",
```
Comment (vmanoharas) [inline certified-connectors/Insiders Cloud/apiDefinition.swagger.json]: I would suggest you keep all parameters descriptions short to fit in the text box. please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#parameters).

File: certified-connectors/Insiders Cloud/apiDefinition.swagger.json [hunk 5]
```diff
@@ -0,0 +1,441 @@
...
+        "Changed",
+        "DBLookup",
+        "AutoFill",
+        "OCR"
+      ]
+    },
+    "FieldCheckState": {
+      "x-nullable": true,
+      "type": "string",
+      "description": "",
```
Comment (vmanoharas) [inline certified-connectors/Insiders Cloud/apiDefinition.swagger.json]: I would suggest you keep all parameters descriptions short to fit in the text box. please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#parameters).

File: certified-connectors/Insiders Cloud/apiDefinition.swagger.json [hunk 6]
```diff
@@ -0,0 +1,441 @@
...
+          "items": {
+            "$ref": "#/definitions/Field"
+          }
+        }
+      }
+    },
+    "DocumentState": {
+      "x-nullable": true,
+      "type": "string",
+      "description": "",
```
Comment (vmanoharas) [inline certified-connectors/Insiders Cloud/apiDefinition.swagger.json]: I would suggest you keep all parameters descriptions short to fit in the text box. please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#parameters).

**Review by vmanoharas — COMMENTED** [review]
Hi @tillbier, 

Kindly resolve review comments.

**Review by vmanoharas — APPROVED** [review]
Dear Partner, 

Congratulations your PR is approved !! We are proud to announce the Brand-New Certification experience for you to certify Power Platform Copilot Connectors & Plugins. Read the blog here- [Announcing Partner Center to certify and publish Power Platform Copilot Connectors and Plugins - Microsoft Power Platform Blog](https://www.microsoft.com/en-us/power-platform/blog/power-automate/announcing-partner-center-to-certify-and-publish-power-platform-copilot-connectors-and-plugins/)

As next steps: 

1.	Learn about the new certification experience- [Get your Power Platform connector and plugin certified - Overview](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Flearn.microsoft.com%2Fen-us%2Fconnectors%2Fcustom-connectors%2Fsubmit-certification&data=05%7C02%7Cv-manoharas%40microsoft.com%7C3fecd63b7d2c4d65d68208dc7bdf57ba%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638521446390456966%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=uWDvs%2BmTbozw1oPc%2FNDNWJ%2FHgnUSfFlRCB1T8UHBh3A%3D&reserved=0)
2.	Package your connector and plugin files- [Prepare Power Platform connector and plugin files for certification](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Flearn.microsoft.com%2Fen-us%2Fconnectors%2Fcustom-connectors%2Fcertification-submission&data=05%7C02%7Cv-manoharas%40microsoft.com%7C3fecd63b7d2c4d65d68208dc7bdf57ba%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638521446390465418%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=pDuv4bnya1eP3Rrs3p4ArNOXw%2FQipWU87q5oZ0VSIe0%3D&reserved=0)
3.	Validate the package for structure before submitting the package for certification to Partner Center - [Run Package Validator tool](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Flearn.microsoft.com%2Fen-us%2Fconnectors%2Fcustom-connectors%2Fcertification-submission%23step-7-validate-the-package-for-structure&data=05%7C02%7Cshefaali.patankar%40microsoft.com%7C6f0c4adf688f42a7107908dcb6c72a8c%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638586213737574171%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=9QGRIRe%2FZ%2B1sgUF5JQjY2DeuyxDt8f8lDwpGFEaaBmg%3D&reserved=0) 
4.	Initiate a certification request in Partner Center- [Verified publisher certification process](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Flearn.microsoft.com%2Fen-us%2Fconnectors%2Fcustom-connectors%2Fsubmit-for-certification&data=05%7C02%7Cv-manoharas%40microsoft.com%7C3fecd63b7d2c4d65d68208dc7bdf57ba%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638521446390471799%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=zbm2px5VZ2ABLBFcORH6tu1fsC9KWdXkP1XJFyluMzw%3D&reserved=0)
5.	Ensure your connector & plugin files comply to the Marketplace policies here- [5000 Power Platform Connector Policies for Marketplace](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Flearn.microsoft.com%2Fen-us%2Flegal%2Fmarketplace%2Fcertification-policies%235000-power-platform-connectors&data=05%7C02%7Cv-manoharas%40microsoft.com%7C3fecd63b7d2c4d65d68208dc7bdf57ba%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638521446390477736%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=h0pI5J5buo82xxvnBVmrAN8K%2F%2BK8xZlUdcSwzWoqWq0%3D&reserved=0) and also [1000 Marketplace policies](https://learn.microsoft.com/en-us/legal/marketplace/certification-policies#100-general)(as applicable) 
6.	Test your connector post certification to provide go-live for deployment- [Testing Guidelines](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Flearn.microsoft.com%2Fen-us%2Fconnectors%2Fcustom-connectors%2Fcertification-testing&data=05%7C02%7Cv-manoharas%40microsoft.com%7C3fecd63b7d2c4d65d68208dc7bdf57ba%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638521446390483743%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=Z8pSnhCArFRzEr373o29qM5h5VbV1nlN2gVbv%2Blk1aE%3D&reserved=0)
7.	If you wish to Update your connector or plugin in future- [Updating Guidelines](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Flearn.microsoft.com%2Fen-us%2Fconnectors%2Fcustom-connectors%2Fcertification-updates&data=05%7C02%7Cv-manoharas%40microsoft.com%7C3fecd63b7d2c4d65d68208dc7bdf57ba%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638521446390489681%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=CvxTWJFE3bTFeJ3nDJoCSpzvimhVV9go4cBG%2FQ4gMvo%3D&reserved=0)

For any queries or concerns please get in touch with us on [certificationteam@microsoft.com](mailto:certificationteam@microsoft.com)

Many thanks,
Power Platform 
Copilot connector & plugin certification team

**Pull request merged**